index.js

```diff
@@ -17,13 +17,31 @@
 
 'use strict';
 
-var forEach = require('for-each');
-var warning = require('warning');
-var has = require('has');
-var trim = require('string.prototype.trim');
+/* Should always be false */
+var debug = false;
+
+var forEach = function (iterable, callback) {
+  var keys = Object.keys(iterable);
+  for (var i = 0; i < keys.length; i += 1) {
+    var key = keys[i];
+    var val = iterable[key];
+    callback(val, key, iterable);
+  }
+};
+
+var has = function (object, property) {
+  return Object.prototype.hasOwnProperty.call(object, property);
+};
+
+var trim = function (input) {
+  return input.replace(/(^\s*)|(\s*$)/mg, '');
```

Comment:
this makes the package vulnerable to someone messing with `String.prototype.replace`, and doesn't properly handle whitespace (in the unicode sense) the way the trim package does.

Reply:
> this makes the package vulnerable to someone messing with `String.prototype.replace`

Who would do that? When I want to kill my JavaScript environment by randomly overriding prototype functions, polyglot.js won't save me…